This replicates on the w530 the patches that zmatt needed to do to get
his aftermarket battery to validate.

Completely untested!
(however, the x230 patch created the same way as this is known to work)

@@ -7343,14 +7343,14 @@
 0001cae0  01 11 42 29 00 02 d8 1e  02 10 d9 1e 42 10 1b ff  |..B)........B...|
 0001caf0  42 28 01 02 da 1e 42 10  db 1e 02 10 c9 70 0a d9  |B(....B......p..|
 0001cb00  3c da 04 db 18 f1 c4 16  01 10 c7 09 5e 80 c7 16  |<...........^...|
-0001cb10  81 10 8c e9 88 e0 ca 21  81 0f 00 00 64 00 f0 0c  |.......!....d...|
+0001cb10  81 10 e0 78 88 e0 ca 21  81 0f 00 00 64 00 f0 0c  |...x...!....d...|
 0001cb20  e1 f1 ca 20 c1 03 0d f0  05 8e 01 e0 0f 78 88 e0  |... .........x..|
 0001cb30  4e 02 2a 00 05 ae 07 d8  c5 03 20 00 04 ae 7f 02  |N.*....... .....|
 0001cb40  20 00 04 8e c9 70 0b d9  eb 05 ef ff 3c da c5 16  | ....p......<...|
 0001cb50  82 10 55 26 43 1c 59 63  58 63 32 89 10 88 00 23  |..U&C.YcXc2....#|
 0001cb60  90 00 08 b9 00 21 04 00  58 63 13 88 59 63 31 89  |.....!..Xc..Yc1.|
 0001cb70  2f 24 08 01 08 b8 38 60  10 78 88 71 da 0d ef f2  |/$....8`.x.q....|
-0001cb80  f8 16 02 11 0b 08 51 00  00 86 83 b8 24 f0 10 10  |......Q.....$...|
+0001cb80  f8 16 02 11 4a 26 00 70  00 86 83 b8 24 f0 10 10  |....J&.p....$...|
 0001cb90  80 20 11 10 81 20 08 b8  38 60 12 10 81 20 10 78  |. ... ..8`... .x|
 0001cba0  13 10 82 20 08 b9 59 61  30 79 e6 0d ef f2 f8 16  |... ..Ya0y......|
 0001cbb0  02 11 0f 08 51 00 00 86  45 20 00 0a 0e f0 05 8e  |....Q...E ......|
@@ -7366,13 +7366,13 @@
 0001cc50  20 00 b2 d1 b0 d0 21 80  14 6a 58 60 14 79 4a 24  | .....!..jX`.yJ$|
 0001cc60  40 74 56 26 c2 16 a8 20  c0 01 01 11 80 04 01 1a  |@tV&... ........|
 0001cc70  12 00 c9 70 0a d9 27 da  bd 04 ef ff 11 db c4 16  |...p..'.........|
-0001cc80  00 10 e1 b8 25 f3 c7 16  80 10 88 e8 e9 70 82 0b  |....%........p..|
+0001cc80  00 10 e1 b8 25 f3 c7 16  80 10 e0 78 e9 70 82 0b  |....%......x.p..|
 0001cc90  ef f1 8a 21 0f 0a 55 f1  05 8e 01 e0 0f 78 69 08  |...!..U......xi.|
 0001cca0  33 02 05 ae 2e f0 c6 0c  ef f1 e9 70 81 e0 11 f5  |3..........p....|
 0001ccb0  47 f1 c9 70 0b d9 7f 04  ef ff 28 da 08 16 c2 10  |G..p......(.....|
 0001ccc0  8c 22 ff 8f 04 f4 17 00  20 00 95 d1 92 d0 34 6a  |."...... .....4j|
 0001ccd0  59 61 3b 79 01 80 11 e0  19 61 62 0f ef fe 56 26  |Ya;y.....ab...V&|
-0001cce0  c0 16 89 e8 00 86 a4 b8  00 a6 14 d8 0f 02 20 00  |.............. .|
+0001cce0  c0 16 e0 78 00 86 a4 b8  00 a6 14 d8 0f 02 20 00  |...x.......... .|
 0001ccf0  04 ae 06 16 c0 10 61 b8  0d 78 0d 08 72 00 06 ae  |......a..x..r...|
 0001cd00  0e d8 f6 f1 00 86 84 b8  f0 f1 a5 ae 05 d8 04 ae  |................|
 0001cd10  c9 70 aa fd e9 01 00 00  21 96 15 d8 9f 09 1e 01  |.p......!.......|
